app/validation/validator.py

```diff
@@ -173,6 +175,29 @@ def validate_duplicates(self, json_to_validate, ignored_keys, special_key):
 
         return duplicate_errors
 
+    def validate_duplicate_options(self, json_to_validate):
+        errors = []
+
+        for block in self._get_blocks(json_to_validate):
+            answers_for_block = self._get_answers_for_block(block)
+
+            for answers in answers_for_block:
+                labels = []
+                values = []
+
+                if 'options' in answers:
+                    for option in answers.get('options'):
```

Comment:
Consider using `for option in answers.get('options', []):` and removing the preceding if statement. Too much nesting makes functions less readable.